src/aics-image-viewer/components/CellViewerCanvasWrapper/index.tsx

```diff
@@ -1,15 +1,15 @@
-import React from "react";
-import { View3d, Volume } from "@aics/volume-viewer";
+import { View3d, Volume } from "@aics/vole-core";
 import { LoadingOutlined } from "@ant-design/icons";
+import React from "react";
 
-import { AxisName, PerAxis, Styles } from "../../shared/types";
```

Comment:
So just some re-ordering here?

Reply:
Yup! Automatic import sorting got added to this repo recently so each time a file is edited for the first time since then, its imports are getting sorted.